lib/config/config.go

```diff
@@ -118,7 +118,7 @@ func (k *Kafka) String() string {
 		k.BootstrapServer, k.GroupID, k.Username != "", k.Password != "")
 }
 
-func (c *Config) TopicConfigs() ([]*kafkalib.TopicConfig, error) {
+func (c Config) TopicConfigs() ([]*kafkalib.TopicConfig, error) {
```

Comment:
Also removed all the pointers from these functions since it's not actually mutating the object